code/modules/transport/tram/tram_remote.dm

```diff
@@ -0,0 +1,146 @@
+#define TRAMCTRL_FAST 1
+#define TRAMCTRL_SAFE 0
+
+/obj/item/assembly/control/transport/remote
+	icon_state = "tramremote_nis"
+	inhand_icon_state = "electronic"
+	lefthand_file = 'icons/mob/inhands/items/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items/devices_righthand.dmi'
+	icon = 'icons/obj/device.dmi'
+	name = "tram remote"
+	desc = "A remote control that can be linked to a tram. This can only go well."
+	w_class = WEIGHT_CLASS_TINY
+	///desired tram destination
+	var/destination
+	///current tram direction
+	var/direction
+	///fast and fun, or safe and boring
+	options = RAPID_MODE
```

Comment:
why is autodoc here it's not being defined here (also move it above the vars you're defining here)